src/firefly/js/templates/lightcurve/LcManager.js

```diff
@@ -273,30 +274,34 @@ export function* lcManager(params={}) {
 }
 
 
-function updateRawTableChart(timeCName, fluxCName, mission='') {
+function updateRawTableChart(timeCName, fluxCName, converterId) {
     var chartX = get(getChartDataElement(LC.RAW_TABLE), ['options', 'x', 'columnOrExpr']);
     var chartY = get(getChartDataElement(LC.RAW_TABLE), ['options', 'y', 'columnOrExpr']);
 
     if (chartX === timeCName && chartY === fluxCName) return;
 
     if (timeCName && fluxCName) {
 
-        const  title = (mission && (mission.toLowerCase()==='wise' || mission.toLowerCase()==='other') )? 'Input Data':'';
+        const rawTable = getTblById(LC.RAW_TABLE);
+
+        //const  title = (mission && (mission.toLowerCase()==='wise' || mission.toLowerCase()==='other') )? 'Input Data':'';
+        const title = getConverter(converterId).showPlotTitle? rawTable.request.META_INFO.title :'';
 
         const xyPlotParams = {x: {columnOrExpr: timeCName}, y: {columnOrExpr: fluxCName, options: 'grid,flip'}, plotTitle:title};
 
         loadXYPlot({chartId: LC.RAW_TABLE, tblId: LC.RAW_TABLE, xyPlotParams, help_id: 'main1TSV.plot'});
     }
 }
 
-function updatePhaseTableChart(flux, mission) {
+function updatePhaseTableChart(flux, converterId) {
     var chartY = get(getChartDataElement(LC.PHASE_FOLDED), ['options', 'y', 'columnOrExpr']);
 
     if (chartY === flux) return;
 
     if (flux) {
 
-        const title= (mission && (mission.toLowerCase()==='wise' || mission.toLowerCase()==='other'))?'Phase Folded Data':'';
+       // const title= (mission && (mission.toLowerCase()==='wise' || mission.toLowerCase()==='other'))?'Phase Folded Data':'';
+        const title = getConverter(converterId).showPlotTitle?'Phase Folded Data':'';
```

Comment:
You see what I mean?  What if other missions want to show something other than 'Phase Folded Data'?